src/App.jsx

```diff
@@ -1,12 +1,10 @@
-import './App.css'
-import HelloWorld from './Components/HelloWorld'
-import UserClick from './Components/CapturingUserClick'
+import React from 'react'
```

Comment:
You no longer need to import React from "react". Starting from the release 17 of React, JSX is automatically transformed without using React. It simplifies the code by removing for unnecessary imports.